index.d.ts

```diff
@@ -0,0 +1,18 @@
+interface ListGithubDirContentOptions {
+	user: string
+	repository: string
+	ref?: string
+	directory: string
+	token: string
```

Comment:
```suggestion
	token?: string
```

Reply:
Got it. Maybe the fact that the token is optional should go into the readme as well.